gitlab/v4/objects/files.py

```diff
@@ -114,9 +114,36 @@ class ProjectFileManager(GetMixin, CreateMixin, UpdateMixin, DeleteMixin, RESTMa
     )
     # NOTE(jlvillal): Signature doesn't match UpdateMixin.update() so ignore
     # type error
```

Comment:
I don't think this comment is needed anymore. And it was slightly wrong as it should have said `GetMixin.get()`